common_features.mk

```diff
@@ -514,6 +514,12 @@ ifeq ($(strip $(GRAVE_ESC_ENABLE)), yes)
     OPT_DEFS += -DGRAVE_ESC_ENABLE
 endif
 
+BSP_DEL_ENABLE ?= yes
+ifeq ($(strip $(BSP_DEL_ENABLE)), yes)
+    SRC += $(QUANTUM_DIR)/process_keycode/process_bsp_del.c
+    OPT_DEFS += -DBSP_DEL_ENABLE
```

Comment:
```suggestion
ifeq ($(strip $(BACKSPACE_DELETE_ENABLE)), yes)
    SRC += $(QUANTUM_DIR)/process_keycode/process_backspace_delete.c
    OPT_DEFS += -DBACKSPACE_DELETE_ENABLE
```

Reply:
`BSP_DEL_ENABLE ?= yes` is dropped here. Is that intentional?

Reply:
This was done for some core features (like Grave Escape) when they were refactored into the `process_record` pattern, so as not to break compatibility since there was previously no enable switch for them. Eventually we'll get around to removing that.